src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -62,21 +84,19 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
     this.runInterval.next(newInterval);
   }
 
-  drawChart(data: FeedbackResponseRecord[], duration: number): void {
-    const svg: any = d3.select('svg');
+  drawChart(canvas: any, data: FeedbackResponseRecord[], duration: number,
```

Comment:
Justify your use of `any` type here.

Reply:
`const canvas: any = d3.select('svg');` i cant find a type for this?

Reply:
Can't verify as I don't have the permission to pull your code.
Let me see after we merge this